JavScript/script.js

```diff
@@ -28,4 +29,63 @@ mobMenuLinks.forEach((link) => {
     mobileMenu.classList.toggle('hidden');
     body.style.overflow = 'auto';
   });
+});
+
+// Create Javascript Objects for Teacher Section
+
+const teacher1 = {
+  image: 'images/yoga_woman_square.jpg',
+  alt: 'Female Yoga Teacher meditating',
+  name: 'Jane Doe',
+  subtitle: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit.',
+  description: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient.',
+};
+
+const teacher2 = {
+  image: 'images/yoga_man_square.jpg',
+  alt: 'Male Yoga Teacher meditating',
+  name: 'John Doe',
+  subtitle: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit.',
+  description: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient.',
+};
+
+const teacher3 = {
+  image: 'images/massage_square.jpg',
+  alt: 'Massage therapist',
+  name: 'Levi Smith',
+  subtitle: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit.',
+  description: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient.',
+};
+
+const teacher4 = {
+  image: 'images/chef_square.jpg',
+  alt: 'Female chef portrait',
+  name: 'Amy Meyer',
+  subtitle: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit.',
+  description: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient.',
+};
```

Comment:
Let us try to match the provided design. Please refer to the original design to implement it. We are missing some designs for the image.

- [ ] Original
![html6](https://user-images.githubusercontent.com/55635977/142241779-7bd7ece7-de25-4e17-ace7-3d823f413d16.png)

- [ ] Yours

![html9](https://user-images.githubusercontent.com/55635977/142261404-c6ece9bd-034b-44cd-8ca6-9bb272d36181.png)

Reply:
Hi Emmyn, 
Thank you for your comment. I have implemented the suggested changes as requested.